[TT-6141] (#2068)
TykTechnologies/tyk-docs: tyk-docs/content/plugins/auth-plugins/id-extractor.md

When a custom authentication mechanism is used, every API call triggers a call to the associated middleware function, if you're using a gRPC-based plugin this translates into a gRPC call. If you're using a native plugin -like a Python plugin-, this involves a Python interpreter call.

> The ID extractor works for some rich plugins: gRPC-based plugins, Python and Lua.

Reviewer: ```suggestion
The ID extractor works the following rich plugins: gRPC-based plugins, Python and Lua.
```
